my-app/src/components/beachspeech/CenterColumn.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import BlobLogo from "../../images/beachspeech/BlobLogo.png"
+
+class CenterColumn extends React.Component {
+    render() {
+        return (
+            <div class="col-5 my-5">   
+                <div class="row">
+                    <img src={BlobLogo} alt="Blob Studios Logo"></img>
+                </div>
+
+                <div class="row offset-1 my-2">
+                    <h2>Blob Studios</h2>
```

Comment:
Theres a couple of "fillers" can we add a `FIXME: replace this with final Logo`
Just so we don't lose track of everything

Same with "Game Art" and the Islands